Resources/ServerInfo/_CP14/Guidebook_EN/JobsTabs/Innkeepers.xml

```diff
@@ -0,0 +1,32 @@
+
+  To keep your crops alive for an extended period, be sure to water them frequently. Keeping plants healthy is easier with the spell **“Plant Growth”** from the Vivification tree, which restores a plant’s water, nutrients, and health.
+
+  If all your seedbeds are full and demand is still high, you can use a hoe to till additional soil. Consider making a seed bag if you plan to expand your farm.
+
+  ## The Cellar
+  The cellar is extremely popular among patrons. Several barrels can be found here that dispense alcohol. Grab a mug, ensure you are drawing from a keg, and profit.
+
+  A typical cellar also contains a few tables and torches, as well as a barrel of water. Tables usually hold spare mugs. Be careful not to serve too much alcohol at once... unless your esteemed customer is a dwarf.
+
+  ## The Kitchen
+
+  Most of an innkeeper's time is spent in the kitchen. First, gather materials: chop or find some wood for planks, start growing crops, locate spare food items, and grab your tools. A good chef will recognize the value of a dagger — typically provided to an innkeeper — which can be used to harvest plants, chop food into slices, and fend off unruly patrons.
+
+  Let’s begin with cooking something simple: a green salad. Start by growing cabbage, pumpkin, and cucumber. Once the crops are ready, bring them into the kitchen. Go to your cooking table and find the recipe for green salad. You may notice the recipe requires sliced ingredients — simply use your knife to prepare them. Next, find a plate (usually some are stored near the ovens). Once all ingredients are near the table, you can begin to assemble the salad.
+
+  Green salad is easy to make because it doesn't require heating. However, almost all other dishes will need to be cooked using the furnace. To use it, place 2–3 wood planks inside to keep the fire burning long enough. Ignite the fire with either the **“Create Flame”** spell or a flint and steel. Then, place the uncooked item on top of the furnace — *after* lighting it — so it begins to cook. Remove the food promptly, or it will burn into inedible ash (which you might try to market, but it won’t end well). The **“Freeze”** spell can stop a furnace from wasting fuel between cooking sessions.
+
+  ## Customers
+  In the kitchen, there is a small ordering window for customers. It is strongly recommended that you create a small menu using paper and a fountain pen, listing the dishes you offer and their prices. Prices are set by the innkeeper, but customers will appreciate affordable options.
+
+  Attract customers with additional services such as bardic music, special deals, or any unique gimmicks you can come up with.
```

Comment:
Don't forget to pay the bard. He'll be pleased, and you can order music!

Also, because of your magnificent "cooking", visitors may vomit. Don't forget to wipe up all dirt and impurities, remove corpses, and then rinse your mop in drinking water.